src/test_l1_dsHdmiIn.c

```diff
@@ -1649,21 +1694,22 @@ void test_l1_dsHdmiIn_negative_dsIsHdmiARCPort_sink(void) {
  * 
  */
 void test_l1_dsHdmiIn_positive_dsGetEDIDBytesInfo_sink(void) {
+
     gTestID = 33;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
     uint8_t numInputPorts = 0;
     unsigned char* edidBytes1 = (unsigned char*) malloc(255);
     unsigned char* edidBytes2 = (unsigned char*) malloc(255);
```

Comment:
unsigned char* edidBytes1 = NULL;
    unsigned char* edidBytes2 = NULL;
    unsigned char* edidBytes3 = NULL;
    unsigned char* edidBytes4 = NULL;
    
        // Step 1: Allocate memory for EDID buffers
    edidBytes1 = (unsigned char*) malloc(255);
    edidBytes2 = (unsigned char*) malloc(255);
    edidBytes3 = (unsigned char*) malloc(255);
    edidBytes4 = (unsigned char*) malloc(255);
    
    do allocations separately